client/app/scripts/components/node-details/node-details-health-link-item.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import NodeDetailsHealthItem from './node-details-health-item';
+import CloudLink from '../cloud-link';
+import { getMetricColor } from '../../utils/metric-utils';
+import { trackMixpanelEvent } from '../../utils/tracking-utils';
+import { GRAPH_VIEW_MODE } from '../../constants/naming';
+
+export default class NodeDetailsHealthLinkItem extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      hovered: false
+    };
+
+    this.onMouseOver = this.onMouseOver.bind(this);
+    this.onMouseOut = this.onMouseOut.bind(this);
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onMouseOver() {
+    this.setState({hovered: true});
+  }
+
+  onMouseOut() {
+    this.setState({hovered: false});
+  }
+
+  onClick() {
+    trackMixpanelEvent('scope.node.metric.click', {
+      layout: GRAPH_VIEW_MODE,
```

Comment:
I was deriving the name from the `scope.node.relative.click` that's in use for a similar purpose. 

The other table mixpanel event `scope.node.metric.click`  is sometimes in the table view and sometimes in the details view. So, technically, we would need to switch there to `scope.node.details.metric.click` if it's within the details. Maybe we can add a property that tells the immediate context/container we are in instead? That would also help to answer the query «do people click on metric links anywhere» w/o the need to combine multiple events.

`'scope.node.metric.click', { context: 'node-details' }`
`'scope.node.metric.click', { context: 'main' }`

In case we want to differentiate further ("it was a table", "it was the status metric section") we can just add more properties. I presume these are filterable in mixpanel. Wdyt?

> Also, the layout could be either table, graph, or resource view, no?

Indeed, will remove.